src/browser/modules/App/keyboardShortcuts.ts

```diff
@@ -59,26 +59,38 @@ function isOutsideTextArea(e: KeyboardEvent): boolean {
 }
 
 export function useKeyboardShortcuts(bus: Bus): void {
-  const trigger = (msg: string): void => bus && bus.send(msg, null)
+  const trigger = useCallback(
+    (msg: string): void => bus && bus.send(msg, null),
+    [bus]
+  )
 
-  const focusEditorOnSlash = (e: KeyboardEvent): void => {
-    if (isOutsideTextArea(e) && matchesShortcut(e, FOCUS_SHORTCUT)) {
-      e.preventDefault()
-      trigger(FOCUS)
-    }
-  }
+  const focusEditorOnSlash = useCallback(
+    (e: KeyboardEvent): void => {
+      if (isOutsideTextArea(e) && matchesShortcut(e, FOCUS_SHORTCUT)) {
+        e.preventDefault()
+        trigger(FOCUS)
+      }
+    },
+    [trigger]
+  )
 
-  const fullscreenEditor = (e: KeyboardEvent): void => {
-    if (
-      matchesShortcut(e, FULLSCREEN_SHORTCUT) ||
-      matchesShortcut(e, OLD_FULLSCREEN_SHORTCUT)
-    ) {
-      e.preventDefault()
-      trigger(EXPAND)
-    }
-  }
+  const fullscreenEditor = useCallback(
+    (e: KeyboardEvent): void => {
+      if (
+        matchesShortcut(e, FULLSCREEN_SHORTCUT) ||
+        matchesShortcut(e, OLD_FULLSCREEN_SHORTCUT)
+      ) {
+        e.preventDefault()
+        trigger(EXPAND)
+      }
+    },
+    [trigger]
+  )
 
-  const keyboardShortcuts = [focusEditorOnSlash, fullscreenEditor]
+  const keyboardShortcuts = useMemo(
+    () => [focusEditorOnSlash, fullscreenEditor],
```

Comment:
I don't really understand what's going on here? Why do we return a list of useCallback hooks?